components/tekton/tasks/get-kubeconfig.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: get-kubeconfig
+spec:
+  description: >-
+    This task can be used to gather kubeconfig file in the output workspace.
```

Comment:
I think it could be nice to add here where the kubeconfig is saved and what are the requirements for running this task (ACM required?).

Reply:
Added description in tekton task.